compiler/rustc_target/src/abi/mod.rs

```diff
@@ -677,32 +677,52 @@ impl Primitive {
     }
 }
 
+/// Inclusive wrap-around range of valid values, that is, if
+/// start > end, it represents `start..=MAX`,
+/// followed by `0..=end`.
+///
+/// That is, for an i8 primitive, a range of `254..=2` means following
+/// sequence:
+///
+///    254 (-2), 255 (-1), 0, 1, 2
+///
+/// This is intended specifically to mirror LLVM’s `!range` metadata,
+/// semantics.
```

Comment:
What does the ", semantics" here do? I can't quite parse this. Did you mean "to mirror the semantics of LLVM’s `!range` metadata"?

Reply:
That whole comment was copied from: https://github.com/rust-lang/rust/pull/88242/files#diff-087cde07a8dbab7f1fb846c8b9367f3fa41381cac27fe06461b68c2e9ed2b623L686-L696
That is the comment for the `valid_range` field in `Scalar`
Idk if it should be kept or not

Reply:
ya drop the comma and it should become comprehensible (and accurate)

Reply:
fixed in #88327